src/HH4b/run_utils.py

```diff
@@ -188,6 +187,13 @@ def parse_common_args(parser):
         default="2022",
         choices=["2018", "2022", "2022EE", "2023", "2023BPix"],
     )
+    parser.add_argument(
+        "--pnet-txbb",
```

Comment:
should we rename this argument since the options are not just pnet anymore

Reply:
done